### PR TITLE
Feat/opentype text fill button

### LINKED ISSUE #75: [Feature] Create new text auto fill option for OpenType Feature
### Feature Description

Create another auto fill text option that clearly displays the possible open type features a language could have instead of the Alice in Wonderland excerpt. A button to reset the text to this new auto fill should also be created.

### Reason for Change

This will allow the OpenType features of a font to clearly be compared and displayed, while also allowing the user to see the effect of enabling and disabling these features (issues #65 #73 #74). 

### Where are the changes needed?

- App.tsx
- App.css
- other files could possibly need editing but most code additions should be minimal

### Code Snippets

_No response_

### Anything Else?

See pull request #70 for similar issue implementation.

### PR DESCRIPTION
**Pull request by calebWei**
### Changes

- Added new button for a different text fill to showcase OTF features
- Made the save to PDF option A3 by default.
- Font features list no longer saves to PDF

### Tests applied

- Works as intended in both Firefox and Chrome

### Issue ticket number(s)

Closes #75 

### Checklist

- [ ] Documented
- [ ] Linting tests successful
- [ ] merge updated prod branch into development branch

### CODE REVIEW

**Review by mwan391 — APPROVED** [review]
Awesome work. Lgtm

**Comment by rhun558** [comment]
@mattysteves There was a conflict issue, just fixed it, will need to be merged now (im not sure if i can because i technically changed the code?)

File: src/components/App.tsx [hunk 1]
```diff
@@ -58,40 +58,47 @@ function App() {
...
-  const setCommonText = () => {
+  const setAliceText = () => {
     const all = document.getElementsByClassName('proof');
     for (const elem of all) {
       elem.textContent = proofingText;
     }
   };
 
+  // Event handler to set a common text for all proof elements (for opentype)
+  const setOpentypeText = () => {
```
Comment (jess-lowe) [inline src/components/App.tsx:102]: Awesome work! This will definitely be necessary now and in the future :)

Comment (jess-lowe) [reply src/components/App.tsx:102]: Maybe it could be changed to incorporate Alice's text through  a parameter reference?

**Pull request merged**